Design Patterns/Abstract Factory Pattern/src/factories/AsusManufacturer.java

```diff
@@ -0,0 +1,18 @@
+package factories;
```

Comment:
please follow naming convention for packaging: like: com.exalt.training.designpattern

please apply it for all